source/Conversation.h

```diff
@@ -93,9 +92,9 @@ class Conversation {
 		// choice can be merged into what came before it, to simplify things.
 		explicit Node(bool isChoice = false) noexcept : isChoice(isChoice), canMergeOnto(!isChoice) {}
 		
-		// For applying condition changes or branching based on conditions:
-		ConditionSet conditions;
-		// For applying actions:
+		// The condition expressions that determine the next node to load.
+		ConditionSet branch;
```

Comment:
Another slight nitpick; The ConditionSet is only part of the branch, the labels/nodes to jump to are also part of branching, so `branchCondition` might be a better name for the variable.

Reply:
Maybe, but to me that falls more into adding the type of the variable to the name of the variable, which I generally feel is redundant.

In some future refactor, where an instantiated `Conversation` exposes only an abstract `Node` interface (rather than `IsAction`, `IsBranch`, etc.), we would have some TU-only private class `Branch` that implements the node interface. That class would better reflect your comment here, in that a branch is not just the tested expressions but also the jump destinations.